gradle/testing/randomization/policies/solr-tests.policy

```diff
@@ -100,6 +100,9 @@ grant {
   permission java.lang.RuntimePermission "loadLibrary.jaas";
   permission java.lang.RuntimePermission "loadLibrary.jaas_unix";
   permission java.lang.RuntimePermission "loadLibrary.jaas_nt";
+  // needed by ONNX integration (TODO: there is a cleaner way to handle this)
```

Comment:
I tried doing the incidivual permissions, and collapsed under the effort ;-). at least this points the way!